website/content/docs/upgrading/upgrade-to-1.15.x.mdx

```diff
@@ -45,6 +45,12 @@ option.
 
 @include 'application-of-sentinel-rgps-via-identity-groups.mdx'
 
+## Auto-roll billing start date
+
+ As of 1.15.13 and later, the billing start date (license start date if not configured) will automatically rollover to the most recent year. 
```

Comment:
Same question here about how, exactly, the date adjusts. The sentence, as written, doesn't really explain what happens.